app/src/main/java/com/codingblocks/cbonlineapp/profile/ReferralActivity.kt

```diff
@@ -14,11 +14,24 @@ import kotlinx.coroutines.launch
 import org.jetbrains.anko.share
```

Comment:
@adiig7 You haven't imported Clipboard and toast, which was lead to a crash for others. Please review your code once before commit.